apps/i18n/common/ja_jp.json

```diff
@@ -530,31 +631,54 @@
   "emptyBlockInVariable": "変数 {name} の中が入力されていません。",
   "emptyBlocksErrorMsg": "”繰り返す”または\"もし\"のブロックを動作をさせるためには内側に別のブロックが必要になります。ブロック内に内側のブロックが適切にはめ込まれているか確認をしてください。",
   "emptyExampleBlockErrorMsg": "関数（かんすう） {functionName} は少（すく）なくとも一つ（ひとつ）の例（れい）が必要（ひつよう）です。それぞれの例（れい）には呼び出し（よびだし）とその結果（けっか）を書（か）いてください。",
+  "emptyFreeResponse": "この質問に対する回答はありません。\n",
   "emptyFunctionBlocksErrorMsg": "関数ブロックは、中に他のブロックがないと動きません。",
   "emptyFunctionalBlock": "入力値が空のブロックがあります。",
+  "emptySection": "あなたのセクションには生徒がいません！ [生徒の管理] ({url}) タブに移動して追加してください。\n",
```

Comment:
Also @castro-jorge with these translations, it looks like there is a lot of broken markdown links

Reply:
Thanks for the heads up, will discuss this with TBH in our meeting tomorrow!